src/weapons.cpp

```diff
@@ -505,20 +505,20 @@ int32_t Weapon::getHealthCost(const Player* player) const
 bool Weapon::executeUseWeapon(Player* player, const LuaVariant& var) const
 {
 	// onUseWeapon(player, var)
-	if (!scriptInterface->reserveScriptEnv()) {
+	if (!tfs::lua::reserveScriptEnv()) {
```

Comment:
"Is there a particular reason for naming it `tfs::lua` instead of simply `lua`

Reply:
@ramon-bernardo avoiding possible clashes with other libs. If at some point we use a dependency that defines a namespace `lua` (say a C++ wrapper for Lua) then it may clash with our functions.

This happened with OpenSSL already (it defines a global `RSA` struct). Ideally, we will wrap the entire code in `namespace tfs` so we can drop `tfs::` and it will be just `lua::`

Reply:
@ranisalt thanks for clarifying, that's what I imagined. We could do a lot like `tfs::x`, right?

I've been looking at game events, I thought something like...
```cpp
tfs::game::events::x
tfs::game::map::tows::x
```